distribution/lib/Standard/Tableau/0.0.0-dev/src/Hyper_Table.enso

```diff
@@ -0,0 +1,75 @@
+from Standard.Base import all
+
+from Standard.Table import Column, Table, Value_Type
+import Standard.Table.Rows_To_Read.Rows_To_Read
+import Standard.Table.Internal.Java_Problems
+
+import project.Hyper_Column.Hyper_Column
+import project.Hyper_File.Hyper_File
+
+polyglot java import java.sql.Types
+polyglot java import org.enso.tableau.HyperReader
+polyglot java import org.enso.tableau.HyperTableColumn
+
+## An Enso representation of a Tableau Hyper Table.
+type Hyper_Table
+    ## PRIVATE
+       Represents a Tableau Hyper Table.
+    private Value file:Hyper_File schema:Text table:Text
+
+    ## PRIVATE
+    to_display_text : Text
+    to_display_text self = self.table + " (" + self.schema + ")"
+
+    ## PRIVATE
+    to_js_object : JS_Object
+    to_js_object self =
+        JS_Object.from_pairs [["type", "Hyper_Table"], ["schema", self.schema], ["table", self.table], ["file", self.file.file.path]]
+
+    ## GROUP Standard.Base.Metadata
+       ICON metadata
+       Reads The Columns for the Table
+    columns : Vector Hyper_Column
+    columns self =
```

Comment:
I'd prefer to call this `column_info` or `column_descriptions` or `column_metadata` or something like that, and the same for the type - it's not a (reference) to a column but just metadata description: name, type. One cannot fetch any data just using `Hyper_Column` instance. At least not now. 

So in that regard as you mentioned, `Hyper_Column` is closer to `Column_Description` than to `DB_Column`.